src/windows/title_editor.py

```diff
@@ -288,7 +288,7 @@ def load_svg_template(self, filename_field=None):
             # Set font size (for possible font dialog)
             s = node.attributes["style"].value
             ard = style_to_dict(s)
-            fs = ard.get("font_size")
+            fs = ard.get("font-size")
```

Comment:
Were you able to confirm this attribute is simply misspelled? Did you find any mention of it in our SVG files? Or in the SVG file format spec?

Reply:
Referring to the latest SVG specifications (https://www.w3.org/TR/SVG2/) > G.2 .Presentation attributes (https://www.w3.org/TR/SVG2/attindex.html#PresentationAttributes), we can see that the correct spelling of the attribute is indeed "font-size". Running `grep -rni "font_size" --include="*.svg"` will return no results, whereas `grep -rni "font-size" --include="*.svg"` will generate results. All the .svg files in the project are using the correct spelling of "font-size" :)